src/theme/Footer/index.tsx

```diff
@@ -108,6 +108,11 @@ function Footer(): JSX.Element | null {
                 <li>
                   <Link to="/">Documentation</Link>
                 </li>
+                <li>
+                  <Link href="https://minaprotocol.com/join-zkignite-cohort-2">
+                    zkIgnite, Cohort 2
+                  </Link>
+                </li>
                 <li>
                   <Link to="/zkapps">Write a zkApp</Link>
```

Comment:
```suggestion
                  <Link to="/zkapps">zkApps Overview</Link>
```

With our links, we want to use short [descriptive link text](https://developers.google.com/style/link-text) that helps provide context for the material that we are linking to.

The https://docs.minaprotocol.com/zkapps link resolves to zkApps Overview, so let's change the link text to match the heading 

If we want to link to https://docs.minaprotocol.com/zkapps/how-to-write-a-zkapp we can change the link to `/zkapps/how-to-write-a-zkapp` and keep the text